initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -302,7 +302,7 @@ configGenCmd() {
     validate_configuration
     configcmd="${configgen_path}/configgen -env ${envfile} -od ${runtime} -id ${componentFile} -c ${compName}"
     log "$configcmd"
-    if [ "${USER}" != "${user}" ]; then
```

Comment:
There are a lot of references to ${USER} in our scripts. Why is this one different to the others? If it isn't, should we be fixing the others too?